python/tvm/relax/block_builder.py

```diff
@@ -365,7 +365,7 @@ def dataflow(self) -> DataflowScope:
         """
         return DataflowScope(self)
 
-    def emit(self, expr: Expr) -> Var:
+    def emit(self, expr: Expr, name_hint: str = "") -> Var:
```

Comment:
to reduce number of conflicts, please send the name_hint parameter change to upstream

Reply:
we don't need to block on upstream being bought up, and can resolve during rebase